docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+## Schema of openstreetmap.by work
+## NOTE: tileserver can be replaced with nginx tile data serving
```

Comment:
yes this is static data serving (updated doc), tileserver-gl is reference implementation of tile server (serve styles, styles data, fonts, data info, tiles) (configured there https://github.com/osm-by/OpenStreetMap.by/pull/13/files#diff-daaf2fceca49ed034eace91c4e875653R69-R76), but it can be replaced with just static data serving (configured https://github.com/osm-by/OpenStreetMap.by/pull/13/files#diff-daaf2fceca49ed034eace91c4e875653R30-R67) as this approach we already use on openstreetmap.by, we still use tileserver in static data preparation there https://github.com/osm-by/OpenStreetMap.by/pull/13/files#diff-af011823155b4051cc327424ccd255f3R151-R176

cons of tileserver:
- as it separate process it can have issues with stability (openstreetmap.by had before)
- it slower that nginx

pros of tileserver:
- it provide more options that nginx as example fonts lists https://github.com/openmaptiles/positron-gl-style/blob/master/style.json#L138 (compare with style we use https://github.com/openmaptiles/osm-bright-gl-style/blob/master/style.json#L1616), data debugging

Reply:
I am sure it is an essential component, but all the links are broken. Created #14 to not forget to document it.